SD_ROOT/wz_mini/wz_mini.conf

```diff
@@ -29,22 +28,10 @@ WIREGUARD_PEER_PUBLIC_KEY=""
 WIREGUARD_PEER_ALLOWED_IPS=""
 WIREGUARD_PEER_KEEP_ALIVE=""
 
-ENABLE_IPTABLES="false"
+ENABLE_IPTABLES="true"
```

Comment:
This change and a few others in `wz_mini.conf` are probably accidental due to conflicts during the rebase?